Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Themes/ToggleBackground.asset

```diff
@@ -12,14 +12,14 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: e7e7db9a2688ed540af9819c456ba2e2, type: 3}
   m_Name: ToggleBackground
   m_EditorClassIdentifier: 
-  Definitions:
+  definitions:
   - ClassName: InteractableColorTheme
     AssemblyQualifiedName: Microsoft.MixedReality.Toolkit.UI.InteractableColorTheme,
       Microsoft.MixedReality.Toolkit.SDK
-    StateProperties:
-    - Name: Color
-      Type: 2
-      Values:
+    stateProperties:
+    - name: Color
```

Comment:
With changes like this....

What if people have crated their own interactable themes? Like MyCustomToggleBackground? If I upgrade, will I have my asset totally broken and need to rebuild it?

Reply:
It should still work because of the FormerlySerializedAs attribute. I have migration code but am still working on the process of what makes sense for User experience. 

Also I have been doing intermittent upgrade paths for each of these PRs. But when I PR the feature branch into MRTK_dev, I am going to create a scene with GA with interactables and themes etc. And then "upgrade" that into the feature branch code

The idea is all stuff should just run as-is because I added the handler at editor and runtime code paths. But I need a way to run a full sweep through the assetdatabase to just fix everything so the files are correct going forward